gobblin-service/src/main/java/org/apache/gobblin/service/modules/flowgraph/datanodes/ExternalDataNode.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.gobblin.service.modules.flowgraph.datanodes;
+
+import com.typesafe.config.Config;
+import org.apache.gobblin.service.modules.dataset.ExternalDatasetDescriptor;
+import org.apache.gobblin.service.modules.flowgraph.BaseDataNode;
+
+
+/**
+ * A DataNode for generic ingress/egress data movement outside of HDFS (HTTP or otherwise)
+ */
+public class ExternalDataNode extends BaseDataNode {
+  public static final String EXTERNAL_PLATFORM_NAME = "external";
```

Comment:
will we have child classes of this generic external datanode? will they override this platform name field?

Reply:
probably not in this case